packages/server/src/services/location-service.js

```diff
@@ -11,22 +14,44 @@ const LocationService = {
 
   async addLocation(props) {
     return db.transaction(async (tx) => {
-      const lastLocation = await tx('locations')
-        .select('id')
+      const unpublishedLocations = await tx('locations')
+        .select('id', 'lat', 'lon')
         .whereNull('published_at')
-        .orderBy('created_at', 'desc')
-        .first();
+        .orderBy('published_at', 'desc');
```

Comment:
Why sort by `published_at` if we're only grabbing the ones that are null?

Reply:
I think I meant to order by `created_at`.

My intention was that it'd check the the oldest records first. But actually now that I think about it, I don't think it'll be needed to do any ordering.